src/main/java/edu/tamu/cap/service/FedoraService.java

```diff
@@ -83,6 +84,9 @@
     public final static String FEDORA_CREATED = "http://fedora.info/definitions/v4/repository#created";
 
     public final static String FEDORA_VERSION = "http://fedora.info/definitions/v4/repository#Version";
+    
+    @Value("${app.messaging.channel}")
+    private String MESSAGING_CHANNEL;
```

Comment:
Not needed here. This file doesn't need any changes.

Reply:
https://github.com/TAMULib/Cap/commit/d420dd8b1aba9565201d3735e9fbb3e6613c6481